### ISSUE TITLE
[BUG] Windows 10 autostart cannot find the file "trakt-scrobbler.vbs" 

### ISSUE THREAD

**Issue by yolorag** [iamkroot/trakt-scrobbler#219]
It's not like super annoying or anything cause I rarely turn my laptop off, but it would be nice if I did not have to manually start it.

I've tried reinstalling both the script, python and mpc-hp.

The file definitely exists.

![Zrzut ekranu (45)](https://user-images.githubusercontent.com/22390622/196438386-23f217aa-1b20-4ddb-b23a-3258d335977e.png)

Win 10 19041
MPC 1.9.23
Python  3.10.8

**Comment by iamkroot** [comment]
> The file definitely exists

How exactly did you verify that?

**Comment by yolorag** [comment]
![Zrzut ekranu (96)](https://user-images.githubusercontent.com/22390622/196517896-d2271875-ee70-479f-9769-b96f81337ff1.png)


I'm sorry, I know nothing about programming lol.

The content of the file is:
"CreateObject("Wscript.Shell").Run "trakts run", 0, False"

Should it be "1, True" ?

**Comment by iamkroot** [comment]
> Should it be "1, True" ?

Not really, those values are as intended (to run the app in the background, instead of creating a window).

Let's try to debug this further-
1. Open up a powershell window ([guide](https://www.howtogeek.com/662611/9-ways-to-open-powershell-in-windows-10/))
2. Run `trakts status`
3. If the above is successful, run `where.exe trakts.exe`. What is the output?

Also, how did you install Python? From your previous screenshot, I'm seeing a non-standard location (`PythonSoftwareFoundation...`)

**Comment by yolorag** [comment]
![user paths](https://user-images.githubusercontent.com/22390622/202010970-3297aa74-a2b8-4ca8-8725-20c5a3c02d63.png)
![PowerShell output](https://user-images.githubusercontent.com/22390622/202010990-6e1bd3ee-8eb1-4f43-9bfd-701ec2bfd9fe.png)


Sorry for delay.
Apparently , for some reason, I have two different versions of Python added to the system path.
I start trakts using old command prompt and it works.

**Comment by iamkroot** [comment]
Ah, this is a PATH issue. The `pipx` folder is not in path for some reason (not sure how cmd is able to find it though).
Try running `python3 -m pipx ensurepath`

**Comment by iamkroot** [comment]
Also try out the suggestions from this thread- https://github.com/pypa/pipx/issues/659

**Comment by yolorag** [comment]
I have tried to update the script and python but everything broke.

I restored my whole system to two weeks old backup.

Then I have realized that trakt.tv API is down lol, but before that I have read your response to similar bug I was experiencing with authentication and you have recommended to downgrade your program to older version.

I found out that it was github client that added the second, not updated version of python to the path.

I'll try to reproduce the bug once trakt API is up again on second computer.
